Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/resources/processdiscoverer/zul/processDiscoverer.zul

```diff
@@ -205,6 +205,7 @@
         <n:span class="ap-action-sep"></n:span>
         <!-- <button ca:data-t="animate" id="animate" w:onClick="Ap.pd.animate()" tooltiptext="Animate" sclass="ap-icon ap-icon-play-circle"/> -->
         <button ca:data-t="animate" id="animate" tooltiptext="Animate" sclass="ap-icon ap-icon-play-circle"/>
+        <button ca:data-t="animate" id="animate_new" tooltiptext="Animate NEW" sclass="ap-icon ap-icon-play-circle"/>
```

Comment:
is it possible to put like some red '*' indicating its a new log annimation. Just a simple quick differ between the 2 log animation. Nothing jazzy.